solvers/heatTransferFoamI/tractionDisplacement/tractionDisplacementFvPatchVectorField.C

```diff
@@ -0,0 +1,224 @@
+/*---------------------------------------------------------------------------*\
```

Comment:
Seems that this code cannot be compiled with the main OpenFOAM library. Moreover, I do not see `Make/options` and `Make/files`.